sdk/formrecognizer/ai-form-recognizer/samples/typescript/src/differentiateLabeledUnlabeled.ts

```diff
@@ -30,14 +30,14 @@ export async function main() {
     throw new Error(`Expecting file ${path} exists`);
   }
 
-  const labeledResponse = await recognizeCustomForm(path, endpoint, apiKey, labeledModelId);
-  const unlabeledResponse = await recognizeCustomForm(path, endpoint, apiKey, unlabeledModelId);
+  const labeledForms = await recognizeCustomForm(path, endpoint, apiKey, labeledModelId);
```

Comment:
following the patter that we used for samples, it will be good to say `forms` and `formsWithLabels`

Reply:
Good point! Fixed.